region/norfair/east.json

```diff
@@ -524,6 +540,46 @@
                     {"heatFrames": 500}
                   ],
                   "devNote": "Does this heat frame count required a midair IBJ? If so, should we have two separate strats?"
+                },
+                {
+                  "name": "Precathedral Sova Reserve Bomb Boost",
+                  "notable": true,
+                  "requires": [
+                    "h_heatProof",
+                    "h_canBombThings",
```

Comment:
I'm assuming PBs won't work. You might need: h_canUseMorphBombs

Reply:
Looking at this again, I am thinking a PB will kill the Sova, and morph bombs will allow IBJ. Is this strat needed?

Reply:
I definitely take two hits, but the important part is 1 hit and the bomb explosion stacking.  The draw of this strat in speedruns is that it can be done using a power bomb.